q2_sample_classifier/tests/test_classifier.py

```diff
@@ -240,6 +241,13 @@ def test_coordinates_format_to_metadata(self):
                         name='Latitude', index=exp_index)
         self.assertEqual(sorted(exp), sorted(obs_category.to_series()))
 
+    # this just checks that palette names are valid input
+    def test_custom_palettes(self):
+        confused = np.array([[1, 0], [0, 1]])
+        for cm in ['Default', *_custom_palettes().keys()]:
```

Comment:
You use `cm` for a confusion matrix elsewhere, could this be renamed to `pallete`?